lib/datadog/core/transport/ext.rb

```diff
@@ -12,6 +12,7 @@ module HTTP
           ADAPTER = Datadog::Core::Configuration::Ext::Agent::HTTP::ADAPTER
           DEFAULT_HOST = Datadog::Core::Configuration::Ext::Agent::HTTP::DEFAULT_HOST
           DEFAULT_PORT = Datadog::Core::Configuration::Ext::Agent::HTTP::DEFAULT_PORT
+          DEFAULT_TIMEOUT_SECONDS = Datadog::Core::Configuration::Ext::Agent::HTTP::DEFAULT_TIMEOUT_SECONDS
```

Comment:
It's weird that we're duplicating these constants here. Maybe consider removing them? After all, the key responsibility of `AgentSettingsResolver` is to provide the correct information in the `AgentSettingsResolver::AgentSettings` object, and no other parts of the code should care about defaults or whatnot.